packages/expo-task-manager/android/src/main/java/expo/modules/taskManager/TaskManagerInternalModule.java

```diff
@@ -57,6 +57,7 @@ public void onCreate(ModuleRegistry moduleRegistry) {
   @Override
   public void onDestroy() {
     mUIManager.unregisterLifecycleEventListener(this);
+    mTaskService.setTaskManager(null, getAppId(), getAppUrl());
```

Comment:
Maybe we can use this new static method `removeTaskManager`? It would be more clear what we're doing here.

Reply:
I've added such method to interface and renamed static one. I guess, whenever we can, we should not use explicit static method. Here, we can use here a method which is publicly available via interface, without assuming which implementation we handle. 

We should rather think of a way not to call this static method anywhere else. I'll think about it.